### PR TITLE
QPPCT-680: empty property warning

### PR DESCRIPTION
**Pull request by clydet**
### Information
- QPPCT-680

### Changes proposed in this PR:
- Added warn logging for empty properties

### Checklist 
- [ ] All JUnit tests pass (`mvn clean verify`).
- [x] New unit tests written to cover new functionality.
- [x] Added and updated JavaDocs for non-test classes and methods.
- [x] No local design debt. Do you feel that something is "ugly" after your changes?
- [ ] Updated documentation (`README.md`, etc.) depending if the changes require it.

### CODE REVIEW

File: commons/src/test/java/gov/cms/qpp/conversion/util/EnvironmentHelperTest.java [hunk 1]
```diff
@@ -26,8 +28,11 @@ static void teardown() {
 	@Test
 	void testIsPresentOnRandomString() {
 		String random = UUID.randomUUID().toString();
+		String message = String.format(EnvironmentHelper.NOT_FOUND, random);
+
 		assertWithMessage("Should not have an environment variable with randomized key")
 				.that(EnvironmentHelper.isPresent(random)).isFalse();
+		assertThat(getLogs()).contains(message);
```
Comment (omnivion) [inline commons/src/test/java/gov/cms/qpp/conversion/util/EnvironmentHelperTest.java]: I think this should be it's own test

File: commons/src/test/java/gov/cms/qpp/conversion/util/EnvironmentHelperTest.java [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
 import java.util.Properties;
 import java.util.UUID;
 
+import gov.cms.qpp.test.logging.LoggerContract;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 
-class EnvironmentHelperTest {
+class EnvironmentHelperTest implements LoggerContract {
```
Comment (halprin) [inline commons/src/test/java/gov/cms/qpp/conversion/util/EnvironmentHelperTest.java]: Sorry, I should have caught this earlier, but we should also have a test for when the env var is present but an empty string.  I believe all the existing tests just use the precondition that the env var doesn't exist.
